pages/lessons/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+import {loadLessons} from '../../lib/lessons'
+
+function LessonCard({lesson}) {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg">
+      <img className="w-full" src={lesson.icon_url} alt={lesson.title}></img>
+      <div className="px-6 py-4">
+        <Link href={`/lessons/[slug]`} as={`/lessons/${lesson.slug}`}>
+          <div className="font-bold text-xl mb-2">
+            <a>{lesson.title}</a>
+          </div>
+        </Link>
+        <p className="text-gray-700 text-base truncate">{lesson.description}</p>
+      </div>
+    </div>
+  )
+}
+
+export default function Courses({lessons}) {
```

Comment:
```suggestion
export default function Lessons({lessons}) {
```

Reply:
On egghead.io as it is now we dropped /lessons in favor of /search with a filter, and I think this is a good move. 5000+ lessons in a paginated list aren't very structurally predisposed to learning. 😂